requirements.txt

```diff
@@ -0,0 +1,6 @@
+pre-commit
```

Comment:
Maybe worth splitting out dev requirements to minimize what is installed on the server